Gekko/UI/MainViewController.swift

```diff
@@ -382,6 +386,13 @@ class MainViewController : UIViewController,
         performSegue(withIdentifier:MainViewController.ShowAccountSettingsSegueName,
                      sender:self)
     }
+    
+    @objc fileprivate func refreshMainView(sender: UIRefreshControl) {
+        chartController.reloadData()
+        ordersStackController.reloadData()
```

Comment:
This logic only forces updating UI controls but actual data is not refreshed. We should get new data from server and updated controls. It may be performed via 'schedule' methods. The function should look like:
scheduleBalanceUpdating()
scheduleOrdersStatusUpdating()
... 
all schedule... methods should be invoked